gunicorn/http/body.py

```diff
@@ -56,8 +57,14 @@ def parse_trailers(self, unreader, data):
 
     def parse_chunked(self, unreader):
         (size, rest) = self.parse_chunk_size(unreader)
+        while_count_size_gt_zero = 0
         while size > 0:
+            print(f"{threading.currentThread().getName()}: size > 0. parse_chunked count={while_count_size_gt_zero}", flush=True)
+            while_count_size_gt_zero += 1
+            while_count_size_gt_len_rest = 0
             while size > len(rest):
+                print(f"{threading.currentThread().getName()}: size > len(rest). parse_chunked count={while_count_size_gt_len_rest}", flush=True)
```

Comment:
can your replace printf by  the debug function from the log handler? This will be more useful.